SPECS/policycoreutils/policycoreutils.spec

```diff
@@ -0,0 +1,5344 @@
+%global libauditver     3.0
+%global libsepolver     2.9-1
```

Comment:
Are these related to this spec's version? If so, I suggest reusing the `%{version}` for their definitions.

Reply:
changed